CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Change Log
 
+## 2023-07-24
+### Added
+- Multi Process Worker Support
```

Comment:
adding this link for context will be helpful - https://docs.fluentd.org/deployment/multi-process-workers

Reply:
Done